CompetitionTemplate/src/competition/subsystems/turret/rotation/commands/RotateTurretWithGamepadCommand.java

```diff
@@ -25,6 +29,13 @@ public void initialize() {
 
     @Override
     public void execute() {
-        rotationSubsystem.setRotationPower(oi.leftJoystick.getButton(8) ? oi.leftJoystick.getRawAxis(4) : 0);
+        double leftIntent = leftTrigger.getVector().y;
+        double rightIntent = -1 * rightTrigger.getVector().y;
+        
+        if(rightIntent != 0 && leftIntent != 0){
```

Comment:
Good notes, the original intention coming into this was that people probably shouldn't be pushing both triggers at the same time and that if they did that we'd probably want to stop the turret. Your point about using small ranges for doubles makes sense.